docker-compose.e2e.yml

```
@@ -0,0 +1,69 @@
+services:
+  indexer:
+    image: oasislabs/oasis-indexer:dev
+    container_name: oasis-indexer
+    depends_on:
+      postgres:
+        condition: service_healthy
+    ports:
+      - 8008:8008
+    command:
+      - --config=/config/e2e-dev.yml
+    volumes:
+      - type: bind
+        source: ./tests/e2e/migrations
+        target: /storage/migrations
+      - type: bind
+        source: ./config
+        target: /config
+      - type: bind
+        source: ./docker/testnet
+        target: /testnet
+  postgres:
+    image: postgres:latest
```

```suggestion
    image: postgres:<some_version>
```
nit: pin this, ideally to what we use in internal deploys

---

Sounds good.